host-contracts/test/tasks/pausing.ts

```diff
@@ -3,18 +3,20 @@ import dotenv from 'dotenv';
 import fs from 'fs';
 import { ethers, run } from 'hardhat';
 
+import { ACL } from '../../typechain-types';
+
 describe('Pausing and Unpausing Tasks', function () {
   let pauserSet;
-  let acl;
+  let acl: ACL;
```

Comment:
Maybe doesn't make much sense to define its type if you then need to do the type assertion `as ACL` in line 19. In such a case, should the pauserSet also have its type defined to be consistent?